repos/spack_repo/builtin/packages/networkdirect/package.py

```diff
@@ -35,6 +35,14 @@ class Networkdirect(msbuild.MSBuildPackage):
     # CBT is entirely deprecated, and fully incompatible with modern dotnet versions
     # so we disable the CBT system and drive the underlying MSBuild system directly
     patch("no_cbt.patch")
+    # MSBuild's native handling of OutputPath does not handle spaces in paths well
```

Comment:
```suggestion
    with when("platform=windows"):
        # MSBuild's native handling of OutputPath does not handle spaces in paths well
```

Reply:
This package can't build anywhere else it's a Windows driver package, `platform=windows` is implied. 

It should have a requires statement if it doesn't.